integration/env/env.go

```diff
@@ -29,6 +38,37 @@ func init() {
 	if clusterName == "" {
 		os.Setenv(EnvClusterName, ClusterName())
 	}
+
+	// azureCDIR must be provided along with other CIDRs,
+	// otherwise we compute CIDRs base on EnvCircleBuildNumber value.
+	azureCDIR := os.Getenv(EnvAzureCIDR)
+	if azureCDIR == "" {
+		buildNumber, err := strconv.ParseUint(os.Getenv(EnvCircleBuildNumber), 10, 32)
+		if err != nil {
+			panic(err)
+		}
+
+		cidrs, err := network.ComputeCIDR(uint(buildNumber))
+		if err != nil {
+			panic(err)
+		}
+
+		os.Setenv(EnvAzureCIDR, cidrs.AzureCIDR)
+		os.Setenv(EnvAzureMasterSubnetCIDR, cidrs.MasterSubnetCIDR)
+		os.Setenv(EnvAzureWorkerSubnetCIDR, cidrs.WorkerSubnetCIDR)
+		os.Setenv(EnvAzureCalicoSubnetCIDR, cidrs.CalicoSubnetCIDR)
```

Comment:
Why setting env vars instead of populating more `var`s (next to cicrcleSHA)?

Reply:
Because those values are later on read from environment by e2e-harness, when templating files under `template` folder.

Reply:
Ack